src/Parse/ParseUser.php

```diff
@@ -383,8 +383,8 @@ public function linkWithTwitter(
         $consumer_key,
         $consumer_secret = null,
         //@codingStandardsIgnoreLine
-        $auth_token,
-        $auth_token_secret,
+        $auth_token = false,
+        $auth_token_secret = false,
```

Comment:
Can you remove the comment `//@codingStandardsIgnoreLine` this is preventing the linter from catching this. Also can you set them to `null` instead of `false`